product_docs/docs/pgd/5/raft/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+
+title: Proxies, Raft and Raft subgroups
+
+---
+
+PGD manages its metadata using a  Raft model where a top-level group spans all the data nodes in the PGD cluster. A Raft leader is elected by the top-level group and propagates the state of the top-level group to all the other nodes in the group. 
+
+!!! Hint What is Raft?
+Raft is an industry accepted algorithm for making decisions though achieving “consensus” from a group of separate nodes within a distributed system. 
+!!!
+
+It is essential for certain operations in the cluster that the Raft leader – such as adding and removing nodes and allocating ranges for [galloc](sequences/#pdg-global-sequences) sequences – is both established and maintained. 
```

Comment:
The "such as" insertion refers the "certain operations", right? Shouldn't it come before "that the Raft leader"?

The word "maintained" seems weird to me. I usually use "operational and connected". With maintenance I connotate manual intervention. But there's nothing for the user to actively do or maintain.

Reply:
maintained as in causing or enabling a state to continue.

changed to
"It is essential for certain operations – such as adding and removing nodes and allocating ranges for [galloc](sequences/#pdg-global-sequences) sequences – in the top-level group that a Raft leader is both established and connected."